source/app/views/gameboard.erb

```diff
@@ -0,0 +1,57 @@
+<div class="row">
+        <div class="col-md-12">
+                <form action='/game' class="form-horizontal" method="post">
+                  <input type='hidden' name='player_1_id' value='<%= user1_id %>' />
+                  <input type='hidden' name='player_2_id' value='<%= user2_id %>' />
+                        <legend class="text-center header"><h1>Play "The Rock", Parchment, Fiskars</h1></legend>
```

Comment:
Instead of having two of these duplicated, we can just send everything we need in and use ruby to loop through game boards for each player!